### ISSUE TITLE
Advance model to RW 7.2.X

### ISSUE THREAD

**Issue by cfelletter** [usbr/MTOM#21]
I saw LC had their hourly model in RW7.2.2. Are we ready to update MTOM?

**Comment by clhorner** [comment]
Yes, we should be ready to update to 7.2.2. I have my most recent development task saved in 7.2.2. Should be done reviewing my changes by tomorrow morning and commit the updates.
